### PR TITLE
Add check for System32 in path before spawning cmd (fix issue #23)

### PR DESCRIPTION
**Pull request by Cosmiiko**
Fixes [this](https://github.com/yibn2008/find-process/issues/23) even though it affects a very small user base.

### CODE REVIEW

**Comment by josevitorsg** [comment]
sup my guy did you fixed you attempting to kill discord error with this? how can do that?

**Comment by Cosmiiko** [comment]
> sup my guy did you fixed you attempting to kill discord error with this? how can do that?

Yeah sort of. In my case it was because of System32 not being in my path.
Type "path" in a command prompt and check if System32 is in there, if it isn't that's your issue too.

**Comment by josevitorsg** [comment]
isn't just java stuffs how can i add system32 to paths? add me on disc? pls Chuu#8649

File: lib/find_process.js [hunk 1]
```diff
@@ -123,7 +123,8 @@ const finders = {
       const cmd = 'WMIC path win32_process get Name,Processid,ParentProcessId,Commandline,ExecutablePath'
       const lines = []
 
-      const proc = utils.spawn('cmd', ['/c', cmd], { detached: false, windowsHide: true })
+      const spawnCmd = process.env.path && process.env.path.includes('C:\\Windows\\System32') ? 'cmd' : 'C:\\Windows\\System32\\cmd.exe'
+      const proc = utils.spawn(spawnCmd, ['/c', cmd], { detached: false, windowsHide: true })
```
Comment (yibn2008) [inline lib/find_process.js:127]: path on windows is case-insensitive, I prefer using regex pattern match instead of `path.includes`.

**Comment by yibn2008** [comment]
Since WMIC has been replaced, this PR is outdated.

**Pull request closed without merging**